python/examples/integrations/writers/Writing_Regression_Performance_Metrics_to_WhyLabs.ipynb

```diff
@@ -0,0 +1,418 @@
+    "\n",
+    "`log_regression_metrics` takes the complete dataframe as input (with input/output features, as well as your prediction column and target column). We also have to define which column is our target (in this case, `temperature`) and which is our prediction column (`prediction_temperature` in this case).\n",
+    "\n",
+    "Once the profile is logged, we can set it's timestamp for the proper day as given by our batch's timestamp.\n",
+    "\n",
+    "Now that we have properly timestamped profiles with regression metrics, we can use the `writer` method to send it to WhyLabs:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "logging data for date 2022-09-07 00:00:00+00:00\n",
+      "writing profiles to whylabs...\n",
+      "logging data for date 2022-09-08 00:00:00+00:00\n",
+      "writing profiles to whylabs...\n",
+      "logging data for date 2022-09-09 00:00:00+00:00\n",
+      "writing profiles to whylabs...\n",
+      "logging data for date 2022-09-10 00:00:00+00:00\n",
+      "writing profiles to whylabs...\n",
+      "logging data for date 2022-09-11 00:00:00+00:00\n",
+      "writing profiles to whylabs...\n",
+      "logging data for date 2022-09-12 00:00:00+00:00\n",
+      "writing profiles to whylabs...\n",
+      "logging data for date 2022-09-13 00:00:00+00:00\n",
+      "writing profiles to whylabs...\n"
+     ]
+    }
+   ],
+   "source": [
+    "from whylogs.api.writer.whylabs import WhyLabsWriter\n",
+    "import whylogs as why\n",
+    "\n",
+    "columns = [\"meta_climate\",\"meta_latitude\",\"meta_longitude\",\"prediction_temperature\",\"temperature\"]\n",
+    "for batch in daily_batches:\n",
+    "    dataset_timestamp = batch.timestamp\n",
+    "\n",
+    "    df = batch.data[columns]\n",
+    "    print(\"logging data for date {}\".format(dataset_timestamp))\n",
+    "    results = why.log_regression_metrics(df, target_column = \"temperature\", prediction_column = \"prediction_temperature\")\n",
+    "\n",
+    "    profile = results.profile()\n",
+    "    profile.set_dataset_timestamp(dataset_timestamp)\n",
+    "    print(\"writing profiles to whylabs...\")\n",
+    "    results.writer(\"whylabs\").write()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can also access directly the regression metrics from your results:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "8.142721906296876"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# other available metrics are: mean_absolute_error and root_mean_squared_error\n",
+    "results.performance_metrics.regression_metrics.mean_squared_error()"
```

Comment:
do we want to surface this to the user?
@jamie256

Reply:
No I think that was more from my testing, you can remove it to keep this concise.

Reply:
ended up removing it. might be confusing since we don't have something similar to classification.
I can add it back later if we think it's worth it!